README.md

```diff
@@ -58,20 +58,67 @@ pip install -U dataprep
 
 ## Examples & Usages
 
-The following examples can give you an impression of what dataprep can do:
+The following examples can give you an impression of what Dataprep can do:
 
 * [Documentation: Connector](https://sfu-db.github.io/dataprep/user_guide/connector/connector.html)
 * [Documentation: EDA](https://sfu-db.github.io/dataprep/user_guide/eda/introduction.html)
 * [EDA Case Study: Titanic](https://sfu-db.github.io/dataprep/user_guide/eda/titanic.html)
 * [EDA Case Study: House Price](https://sfu-db.github.io/dataprep/user_guide/eda/house_price.html)
 
+### Connector
+
+Connector is an intuitive, open-source API wrapper that speeds up development by standardizing calls to multiple APIs as a simple workflow.
+
+
+Connector provides a simple programming interface to collect structured data from different Web APIs (e.g., Twitter API, Yelp Fusion API, Spotify API, DBLP API), making web data collection easy and efficient, without requiring advanced programming skills.
+
+
+Do you want to leverage the growing number of websites that are opening their data through public APIs?
+
+
+Connector is for you!  
+
+Let's check out the several benefits that Connector offers:
+
+* <ins>**A unified API:**</ins> You can fetch data using one or two lines of code to get data from many websites.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://sfu-db.github.io/dataprep/user_guide/connector/connector.html"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/connector_main.gif"/></a>
+
+* <ins>**Auto Pagination:**</ins> Connector automatically does the pagination for you! Just specify the desired number of returned results (argument `_count`) without getting into unnecessary detail about a specific pagination scheme.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://sfu-db.github.io/dataprep/user_guide/connector/connector.html"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/connector_pagination.gif"/></a>
+
+* <ins>**Smart API request strategy:**</ins> Through the `_concurrency` argument, Connector simplifies concurrency, issuing API requests in parallel while respecting the API's rate limit policy. 
```

Comment:
Also propose we explain the value of this so beginners don't have to read between the lines to understand the value of this feature.

Reply:
Done. The following sentence was included: "Do you want to fetch results more quickly by making concurrent requests to Web APIs?".